### PR TITLE
Adding version env var

### PR DESCRIPTION
**Pull request by zaelgohary**
Related issue: https://github.com/threefoldtech/tfgrid_dashboard/issues/284

### CODE REVIEW

File: .github/workflows/Docker.yaml [hunk 1]
```diff
@@ -1,29 +0,0 @@
-name: Build and push a new docker image based on project release tag.
```
Comment (AhmedHanafy725) [inline .github/workflows/Docker.yaml]: Why did this file got deleted?

File: Dockerfile [hunk 2]
```diff
@@ -28,4 +28,4 @@ RUN chmod +x build-env.sh
 
 # Serve the app
 EXPOSE 80
-CMD [ "/bin/bash", "-c", "/usr/share/nginx/html/build-env.sh && nginx -g \"daemon off;\"" ]
+CMD [ "/bin/bash", "-c", "/usr/share/nginx/html/build-env.sh && nginx -g \"daemon off;\"" ]
```
Comment (AhmedHanafy725) [inline Dockerfile]: no changes should be done in this file

File: public/config.js [hunk 3]
```diff
@@ -1 +1 @@
...
+window.configs = { APP_API_URL: 'wss://tfchain.dev.grid.tf/ws', APP_STELLAR_HORIZON_URL: 'https://horizon-testnet.stellar.org', APP_TFT_ASSET_ISSUER: 'GA47YZA3PKFUZMPLQ3B5F2E3CJIB57TGGU7SPCQT2WAEYKN766PWIMB3', APP_BRIDGE_TFT_ADDRESS: 'GDHJP6TF3UXYXTNEZ2P36J5FH7W4BJJQ4AYYAXC66I2Q2AH5B6O6BCFG', APP_ACTIVATION_SERVICE_URL: 'https://activation.dev.grid.tf', APP_EXPLORER_URL: 'https://explorer.dev.grid.tf/', APP_GRAPHQL_URL: 'https://graphql.dev.grid.tf/graphql', APP_GRIDPROXY_URL: 'https://gridproxy.dev.grid.tf', APP_NETWORK: 'dev', APP_VERSION: 'dev1.0.0', APP_GRAVITY_CONTRACT_ADDRESS: '0xBF8C35Ad93366E159C0F0B75F1a4f4ad6Ff80811', APP_TFT_TOKEN_CONTRACT_ADDRESS: '0xDC5a9199e2604A6BF4A99A583034506AE53F4B34', APP_BRIDGE_FEES: '3', APP_TFT_DECIMALS: 7, APP_TFT_DENOM: 'TFT', APP_PROPOSAL_DENOM: 'TFT', APP_COSMOS_REST: 'https://tfhub.test.grid.tf:1317/', APP_TENDERMINT_RPC: 'https://tfhub.test.grid.tf:26657/', APP_GAS_PRICE: '80TFT', APP_CHAIN_ID: 'threefold-hub-testnet', PLAYGROUND_URL: 'https://play.dev.grid.tf', };
```
Comment (AhmedHanafy725) [inline public/config.js]: here too

File: src/Dashboard.vue [hunk 4]
```diff
@@ -339,7 +343,8 @@ export default class Dashboard extends Vue {
     this.$store.dispatch("portal/subscribeAccounts");
     this.balanceFree = this.$route.query.balanceFree;
     this.balanceReserved = this.$route.query.balanceReserved;
-
+    console.log("version", this.version);
```
Comment (AhmedHanafy725) [inline src/Dashboard.vue]: please remove this log

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

File: src/Dashboard.vue (hunk 4 above)
Comment (zaelgohary) [reply src/Dashboard.vue]: done.

File: .github/workflows/Docker.yaml (hunk 1 above)
Comment (zaelgohary) [reply .github/workflows/Docker.yaml]: It was deleted by mistake. I've restored it now.

File: public/config.js (hunk 3 above)
Comment (zaelgohary) [reply public/config.js]: done.

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**